### PR TITLE
Atualização do readme

### PR DESCRIPTION
**Pull request by Eduardolimr**
## Descrição
Atualizar readme.

## Porque este Pull Request é necessário?
Para que o readme esteja mais condizente com a wiki do projeot.

## Critérios de Aceitação
- [x] Readme com mais informações
- [x] Readme com fotos

### CODE REVIEW

**Comment by Eduardolimr** [comment]
Ainda tenho que mexer nas imagens pois quebraram depois de eu ter migrado para o readme.

File: README.md [hunk 1]
```diff
@@ -18,46 +25,76 @@
...
 ```
 npm run test:unit
 ```
 
+# Membros
+
+## Equipe de EPS
+||**Membros**|**E-mail**|**GitHub**|
+|:-:|:-:|:-:|:-:|
+|<img src=" https://i.ibb.co/4gqXmYg/eduardolima.png" width="100" height="100"/>|Eduardo Lima|eduardolimrib@gmail.com|[@Eduardolimr](https://github.com/Eduardolimr)|
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

**Comment by lucianosz7** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 13
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/2019.2-ArBC/2019.2-ArBC/pullRequest?prid=4488266&bid=15140065)

File: README.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 
 ![](https://jlucassr.github.io/ArBC-Pages/imagens/logo.jpg)
 
+## Porquê do Nome?
+- “Ar” vem de um acrônimo para Augmented Reality (AR), realidade aumentada em inglês.
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 3]
```diff
@@ -18,46 +25,76 @@
...
+
+## Equipe de MDS
+
+||**Membros**|**E-mail**|**GitHub**|
+|:-:|:-:|:-:|:-:|
+|<img src=" https://i.ibb.co/s9Vr8qc/igor.png" width="100" height="100"/>|Igor Batista|igorbatistapaiva@outlook.com|[@igor-paiva](https://github.com/igor-paiva)|
+|<img src=" https://i.ibb.co/Wft4bC6/joaohenrique.png" width="100" height="100"/>|João Henrique|joao.henrique1299@gmail.com|[@JoaoHenrique12](https://github.com/JoaoHenrique12)|
+|<img src=" https://i.ibb.co/0X55hLW/marcelo.png" width="100" height="100"/>|Marcelo Victor|marcelovictorg2@gmail.com|[@marcelog5](https://github.com/marcelog5)|
+|<img src=" https://i.ibb.co/mhCz5gb/rhuan.png" width="100" height="100"/>|Rhuan Carlos|rhuancarlos.queiroz@gmail.com|[@Rhuancpq](https://github.com/Rhuancpq)|
+|<img src=" https://i.ibb.co/2P6p1Vx/sergio.png" width="100" height="100"/>|Sérgio Almeida|sergiosacj@hotmail.com.br|[@SergioAlmeidaCiprianoJr](https://github.com/SergioAlmeidaCiprianoJr)|
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 4]
```diff
@@ -18,46 +25,76 @@
...
 npm run test:unit
 ```
 
+# Membros
+
+## Equipe de EPS
+||**Membros**|**E-mail**|**GitHub**|
+|:-:|:-:|:-:|:-:|
+|<img src=" https://i.ibb.co/4gqXmYg/eduardolima.png" width="100" height="100"/>|Eduardo Lima|eduardolimrib@gmail.com|[@Eduardolimr](https://github.com/Eduardolimr)|
+|<img src=" https://i.ibb.co/xGd3zdH/joaolucas.png" width="100" height="100"/>|João Lucas|joao.lucas.ssr@gmail.com|[@jlucassr](https://github.com/jlucassr)|
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 5]
```diff
@@ -18,46 +25,76 @@
...
+
+
+## Equipe de MDS
+
+||**Membros**|**E-mail**|**GitHub**|
+|:-:|:-:|:-:|:-:|
+|<img src=" https://i.ibb.co/s9Vr8qc/igor.png" width="100" height="100"/>|Igor Batista|igorbatistapaiva@outlook.com|[@igor-paiva](https://github.com/igor-paiva)|
+|<img src=" https://i.ibb.co/Wft4bC6/joaohenrique.png" width="100" height="100"/>|João Henrique|joao.henrique1299@gmail.com|[@JoaoHenrique12](https://github.com/JoaoHenrique12)|
+|<img src=" https://i.ibb.co/0X55hLW/marcelo.png" width="100" height="100"/>|Marcelo Victor|marcelovictorg2@gmail.com|[@marcelog5](https://github.com/marcelog5)|
+|<img src=" https://i.ibb.co/mhCz5gb/rhuan.png" width="100" height="100"/>|Rhuan Carlos|rhuancarlos.queiroz@gmail.com|[@Rhuancpq](https://github.com/Rhuancpq)|
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 6]
```diff
@@ -18,46 +25,76 @@
...
+|<img src=" https://i.ibb.co/4gqXmYg/eduardolima.png" width="100" height="100"/>|Eduardo Lima|eduardolimrib@gmail.com|[@Eduardolimr](https://github.com/Eduardolimr)|
+|<img src=" https://i.ibb.co/xGd3zdH/joaolucas.png" width="100" height="100"/>|João Lucas|joao.lucas.ssr@gmail.com|[@jlucassr](https://github.com/jlucassr)|
+|<img src=" https://i.ibb.co/NxTMn7m/lucianosantos.png" width="100" height="100"/>|Luciano Santos|Luciano_z7@hotmail.com|[@lucianosz7](https://github.com/lucianosz7)|
+
+
+## Equipe de MDS
+
+||**Membros**|**E-mail**|**GitHub**|
+|:-:|:-:|:-:|:-:|
+|<img src=" https://i.ibb.co/s9Vr8qc/igor.png" width="100" height="100"/>|Igor Batista|igorbatistapaiva@outlook.com|[@igor-paiva](https://github.com/igor-paiva)|
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 7]
```diff
@@ -18,46 +25,76 @@
...
 ```
 
+# Membros
+
+## Equipe de EPS
+||**Membros**|**E-mail**|**GitHub**|
+|:-:|:-:|:-:|:-:|
+|<img src=" https://i.ibb.co/4gqXmYg/eduardolima.png" width="100" height="100"/>|Eduardo Lima|eduardolimrib@gmail.com|[@Eduardolimr](https://github.com/Eduardolimr)|
+|<img src=" https://i.ibb.co/xGd3zdH/joaolucas.png" width="100" height="100"/>|João Lucas|joao.lucas.ssr@gmail.com|[@jlucassr](https://github.com/jlucassr)|
+|<img src=" https://i.ibb.co/NxTMn7m/lucianosantos.png" width="100" height="100"/>|Luciano Santos|Luciano_z7@hotmail.com|[@lucianosz7](https://github.com/lucianosz7)|
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 8]
```diff
@@ -2,6 +2,13 @@
 
 ![](https://jlucassr.github.io/ArBC-Pages/imagens/logo.jpg)
 
+## Porquê do Nome?
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 9]
```diff
@@ -18,46 +25,76 @@
...
+|<img src=" https://i.ibb.co/xGd3zdH/joaolucas.png" width="100" height="100"/>|João Lucas|joao.lucas.ssr@gmail.com|[@jlucassr](https://github.com/jlucassr)|
+|<img src=" https://i.ibb.co/NxTMn7m/lucianosantos.png" width="100" height="100"/>|Luciano Santos|Luciano_z7@hotmail.com|[@lucianosz7](https://github.com/lucianosz7)|
+
+
+## Equipe de MDS
+
+||**Membros**|**E-mail**|**GitHub**|
+|:-:|:-:|:-:|:-:|
+|<img src=" https://i.ibb.co/s9Vr8qc/igor.png" width="100" height="100"/>|Igor Batista|igorbatistapaiva@outlook.com|[@igor-paiva](https://github.com/igor-paiva)|
+|<img src=" https://i.ibb.co/Wft4bC6/joaohenrique.png" width="100" height="100"/>|João Henrique|joao.henrique1299@gmail.com|[@JoaoHenrique12](https://github.com/JoaoHenrique12)|
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 10]
```diff
@@ -18,46 +25,76 @@
...
+|<img src=" https://i.ibb.co/0X55hLW/marcelo.png" width="100" height="100"/>|Marcelo Victor|marcelovictorg2@gmail.com|[@marcelog5](https://github.com/marcelog5)|
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 11]
```diff
@@ -18,46 +25,76 @@
...
+
+## Equipe de EPS
+||**Membros**|**E-mail**|**GitHub**|
+|:-:|:-:|:-:|:-:|
+|<img src=" https://i.ibb.co/4gqXmYg/eduardolima.png" width="100" height="100"/>|Eduardo Lima|eduardolimrib@gmail.com|[@Eduardolimr](https://github.com/Eduardolimr)|
+|<img src=" https://i.ibb.co/xGd3zdH/joaolucas.png" width="100" height="100"/>|João Lucas|joao.lucas.ssr@gmail.com|[@jlucassr](https://github.com/jlucassr)|
+|<img src=" https://i.ibb.co/NxTMn7m/lucianosantos.png" width="100" height="100"/>|Luciano Santos|Luciano_z7@hotmail.com|[@lucianosz7](https://github.com/lucianosz7)|
+
+
+## Equipe de MDS
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 12]
```diff
@@ -18,46 +25,76 @@
...
+
+||**Membros**|**E-mail**|**GitHub**|
+|:-:|:-:|:-:|:-:|
+|<img src=" https://i.ibb.co/s9Vr8qc/igor.png" width="100" height="100"/>|Igor Batista|igorbatistapaiva@outlook.com|[@igor-paiva](https://github.com/igor-paiva)|
+|<img src=" https://i.ibb.co/Wft4bC6/joaohenrique.png" width="100" height="100"/>|João Henrique|joao.henrique1299@gmail.com|[@JoaoHenrique12](https://github.com/JoaoHenrique12)|
+|<img src=" https://i.ibb.co/0X55hLW/marcelo.png" width="100" height="100"/>|Marcelo Victor|marcelovictorg2@gmail.com|[@marcelog5](https://github.com/marcelog5)|
+|<img src=" https://i.ibb.co/mhCz5gb/rhuan.png" width="100" height="100"/>|Rhuan Carlos|rhuancarlos.queiroz@gmail.com|[@Rhuancpq](https://github.com/Rhuancpq)|
+|<img src=" https://i.ibb.co/2P6p1Vx/sergio.png" width="100" height="100"/>|Sérgio Almeida|sergiosacj@hotmail.com.br|[@SergioAlmeidaCiprianoJr](https://github.com/SergioAlmeidaCiprianoJr)|
+|<img src=" https://i.ibb.co/741s3JW/thiago.png" width="100" height="100"/>|Thiago Santos|thiago.lopes.santos.tls@gmail.com|[@thiagolopess](https://github.com/thiagolopess)|
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md [hunk 13]
```diff
@@ -2,6 +2,13 @@
 
 ![](https://jlucassr.github.io/ArBC-Pages/imagens/logo.jpg)
 
+## Porquê do Nome?
+- “Ar” vem de um acrônimo para Augmented Reality (AR), realidade aumentada em inglês.
+- “ABC” vem como um símbolo para alfabetização.
```
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

**Comment by lucianosz7** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 13
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/2019.2-ArBC/2019.2-ArBC/pullRequest?prid=4488266&bid=15140065)

File: README.md (hunk 1 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 2 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 3 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 4 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 5 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 6 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 7 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 8 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 9 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 10 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 11 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 12 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 13 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 1 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 2 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 3 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 4 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 5 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 6 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 7 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 8 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 9 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 10 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 11 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[no-consecutive-blank-lines] Remove 1 line before node](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 12 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[table-pipe-alignment] Misaligned table fence](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

File: README.md (hunk 13 above)
Comment (lucianosz7) [inline README.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/app/ArBC/2019.2-ArBC/pullRequest?prid=4488266)

**Pull request merged**